src/components/ValidationStatus.vue

```diff
@@ -215,6 +127,7 @@ $button-color: #3BD7FF;
 }
 
 .validation-container {
+  cursor: default;
```

Comment:
Previously the cursor implied that something in the validation panel could be clicked. This changes that and makes sure the default cursor is displayed.